.github/workflows/push.yml

```diff
@@ -90,7 +89,9 @@ jobs:
           if [ $branch == "master" ] || [ ${array[1]} == "tags" ] || [[ $branch =~ ^v[0-9]*.[0-9]*.x$ ]]
           then
             source env-vars
-            docker push ${{ secrets.REPONAME }}/${GRAPHQL_SERVER_IMAGE}:${IMG_TAG}
+            cd litmus-portal && make docker.buildx
+            cd graphql-server
+            docker buildx build --file build/Dockerfile --progress plane --push --no-cache --platform linux/arm64,linux/amd64 --tag ${{ secrets.REPONAME }}/${GRAPHQL_SERVER_IMAGE}:${IMG_TAG} .
```

Comment:
Can you use docker buildx GitHub action? The command looks very messy.

Reply:
Have simplified it, re-used a single command from `Makefile` for building diff component images. We are building binaries inside Dockerfile so we are accessing different directories and building images which is difficult to do with actions.